src/Uploader/UploaderInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+namespace TYPO3\Darth\Uploader;
+
+/*
+ * This file is part of the TYPO3 project.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+interface UploaderInterface
```

Comment:
do we need the interface at all if we don't use it really?

Reply:
Well, it's used by AzureUploader and EltsUploader, which allows me to separate the uploaders